src/analysis/souffle/sensitivity_analysis.dl

```diff
@@ -112,13 +95,29 @@ sqlValueSensitivity(value, sensitivity_score) :-
       sqlValueHasSensitivityScoreAtIndex(value, input_sensitivity, input_list_length-1), 
       sensitivity_score = input_sensitivity * function_sensitvity.
 
+.decl isGlobalEpsilon(epsilon: number)
+//TODO: is there a way to set a deafault option?
+isGlobalEpsilon(epsilon) :- isOperation([_, "global_epsilon", _, _, [["epsilon", $NumberAttributePayload(epsilon)], _]]).
+
+.decl isDiffPrivacyMechanism(value: AccessPath, epsilon: number)
+isDiffPrivacyMechanism(value, epsilon) :- isOperation([_, "privacy_mechanism", value, _, [["epsilon", $NumberAttributePayload(epsilon)], _]]). 
+
+//sequential composition
+.decl epsilonDiffPrivacy(value: AccessPath, epsilon: number)
+epsilonDiffPrivacy(value, epsilon) :- isDiffPrivacyMechanism(value, _),
+      epsilon = sum y : { isDiffPrivacyMechanism(value, y)}. 
+
+violatesPolicy(value) :- isSqlOperator([_, _, value, _, _]), 
```

Comment:
We should probably only check for value that satisfy `isSqlOutput`?

Reply:
We currently do not have such a relation/fact. What would that model (that is not currently modeled by the output of an Operation)?

Reply:
I just saw the relation defined in another file. I think with the new isAggregationOperation relation this would no longer be  necessary, right?